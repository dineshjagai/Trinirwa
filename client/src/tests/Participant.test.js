import React from 'react';
import renderer from 'react-test-renderer';
import Participant from '../live_stream/Participant';

test('Test Participant snapshot', () => {
  const part = {
      'videoTracks': 1,
      'audioTracks': 1,
      'identity': 'isimbib',
  };
  const component = renderer.create(<Participant participant={part}/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
