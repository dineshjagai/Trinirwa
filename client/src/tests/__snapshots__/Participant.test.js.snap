// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Participant snapshot 1`] = `
<div
  className="participant"
>
  <h3>
    isimbib
  </h3>
  <video
    autoPlay={true}
  />
  <audio
    autoPlay={true}
    muted={true}
  />
</div>
`;
